packages/mdc-data-table/component.ts

```diff
@@ -135,10 +135,14 @@ export class MDCDataTable extends MDCComponent<MDCDataTableFoundation> {
     this.headerRow.removeEventListener('click', this.headerRowClickListener);
     this.content.removeEventListener('change', this.handleRowCheckboxChange);
 
-    this.headerRowCheckbox.destroy();
-    this.rowCheckboxList.forEach((checkbox) => {
-      checkbox.destroy();
-    });
+    if (this.headerRowCheckbox) {
```

Comment:
Thanks for sending the pull request! Can you also update `initialSyncWithDOM()` method to add event listener only when row checkboxes are present?